### DISCUSSION TITLE
list-item-spacing should not exist

### DISCUSSION THREAD

**Discussion by Mister-Hope** [remarkjs/remark#550] [Q&A] [answered]
I don't agree with this rule as it's useless.

People can create a list of `<li><p></p></li>` to make their list more sperated.

Here we can create a list of items which are hightlighted:

```md
- **Something A which is long but werid if I break it somewhere**

- **Something B which is long but werid if I break it somewhere**

- **Something C which is long but werid if I break it somewhere**

```

- **Something A which is long but werid if I break it somewhere**

- **Something B which is long but werid if I break it somewhere**

- **Something C which is long but werid if I break it somewhere**

I am leaving blanks to create a `<p>` tag, which makes the list not tight! People can expect that in certain situations! why should the rule stop this? @wooorm

I don't want this:

- **Something A which is long but werid if I break it somewhere**
- **Something B which is long but werid if I break it somewhere**
- **Something C which is long but werid if I break it somewhere**

And I think it is totally werid to make this work around:

```md
- **Something A which is long but werid**
  **if I break it somewhere**

- **Something B which is long but werid if I break it somewhere**

- **Something C which is long but werid if I break it somewhere**
```

- **Something A which is long but werid**
  **if I break it somewhere**

- **Something B which is long but werid if I break it somewhere**

- **Something C which is long but werid if I break it somewhere**

**Comment by ChristianMurphy** [comment]
@Mister-Hope I'm not sure I follow.
`list-item-spacing` allows you to select between a tight list or loose list https://spec.commonmark.org/0.29/#lists
It is a valid rule.

> I don't agree with this rule as it's useless.

If you don't want the rule don't enable it.

**Comment by wooorm** [comment, marked as answer]
All remark lint rules can be turned off. They’re opt in.

@Mister-Hope Please stop spamming and stop using such unprofesional language. [Here’s what you get for free](https://github.com/remarkjs/remark-lint/blob/main/license). See the [code of conduct](https://github.com/remarkjs/.github/blob/HEAD/code-of-conduct.md) and adhere to it or you’ll be banned.

**Reply by Mister-Hope** [reply]
> unprofesional 

What's this? So everyone on github has to be a native English speaker do they?

**Reply by ChristianMurphy** [reply]
@Mister-Hope unprofessional in this context means verbally attacking the project and maintainers ([ad hominem](https://zh.wikipedia.org/wiki/%E8%A8%B4%E8%AB%B8%E4%BA%BA%E8%BA%AB)), rather than asking earnest questions about how the project works, or offering suggestions towards how it could be improved.

**Reply by ChristianMurphy** [reply]
also as stated above and in your other issues, all remark lint rules are optional, if you don't like or disagree with a rule, that's fine! It can be excluded or disabled in a project.
`list-item-spacing` is a stylistic rule, disagreeing with the style is fine, but it also doesn't make it "wrong" or "useless".
